db/schema.rb

```diff
@@ -104,6 +112,17 @@
     t.string "source"
   end
 
+  create_table "appeals_api_status_updates", force: :cascade do |t|
+    t.string "from"
+    t.string "to"
+    t.string "statusable_type"
+    t.string "statusable_id"
+    t.datetime "status_update_time"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
```

Comment:
Not sure we need all these timestamps. `status_update_time` is a copy of `created_at`, right? And we shouldn't ever be updating these so `updated_at` seems redundant, but since it's part of the rails default timestamps I don't mind it staying.

What about using `created_at` instead, and if necessary aliasing `status_update_time` to `created_at`?

Reply:
I think the idea is that the event propagation can be eventually consistent. `status_update_time` is the generated timestamp at the time of the actual update, whereas the created_at and updated_at (as you said) are the default timestamps that relate to the audit log record itself.